README.md

```diff
@@ -88,35 +88,8 @@ To use the Docker image, run:
 
 ### pre-commit
 
-It is possible to use shfmt with [pre-commit][pre-commit] and a `local`
-repo configuration like:
-
-```yaml
-  - repo: local
-    hooks:
-      - id: shfmt
-        name: shfmt
-        minimum_pre_commit_version: 2.4.0
-        language: golang
-        additional_dependencies: [mvdan.cc/sh/v3/cmd/shfmt@v3.2.2]
-        entry: shfmt
-        args: [-w]
-        types: [shell]
-```
-
-An alternative configuration for running shfmt via the official shfmt docker image is:
-
-```yaml
-  - repo: local
-    hooks:
-      - id: shfmt
-        name: shfmt
-        minimum_pre_commit_version: 2.4.0
-        language: docker_image
-        types: [shell]
-        entry: mvdan/shfmt:v3.4.0
-        args: [-w]
-```
+[pre-commit][pre-commit] hook for shfmt is available at
+https://github.com/scop/pre-commit-shfmt
```

Comment:
make this a link just like pre-commit above, e.g. pre-commit-shfmt

Reply:
Done; plain `pre-commit-shfmt` as the link text would read a bit strange to me, so used `scop/pre-commit-shfmt`.

BTW, now that it's a oneliner, I think it would fit in fine in the "other noteworthy integrations" section, no need for a separate section of its own any more. Let me know if you agree and I'll make it so.

Reply:
That's a good idea :)

Reply:
And done :)